src/gui/input.h

```diff
@@ -5,14 +5,15 @@
 #include <QMap>
 #include <QPoint>
 #include <QString>
+#include <QLocale>
 
 namespace NeovimQt { namespace Input {
 
 /// Returns the QMap of all NeoVim-recognized special keys.
 const QMap<int, QString>& GetSpecialKeysMap() noexcept;
 
 /// Convert Qt key input into Neovim key-notation. See QKeyEvent.
-QString convertKey(const QKeyEvent& ev) noexcept;
+QString convertKey(const QKeyEvent& ev, QLocale* locale = nullptr) noexcept;
```

Comment:
`const QLocale*` (if possible).

Sorry, my bad I missed this in my API suggestion.

Reply:
done